logging/src/logging.cc

```diff
@@ -53,7 +53,7 @@ namespace leatherman { namespace logging {
                 << expr::format_date_time<boost::posix_time::ptime>("TimeStamp", "%Y-%m-%d %H:%M:%S.%f")
                 << " " << left << setfill(' ') << setw(5) << leatherman::logging::log_level_attr
                 << " " << leatherman::logging::namespace_attr
-                << " - " << expr::smessage);
+                << ":" << expr::smessage);
```

Comment:
If we add a macro to disable printing the line number, I'd move the `:` formatting to line 158.